src/parkapi_sources/converters/kienzler/models.py

```diff
@@ -45,3 +57,60 @@ def to_realtime_parking_site(self) -> RealtimeParkingSiteInput:
             realtime_capacity=self.sum_boxes,
             realtime_free_capacity=self.bookable,
         )
+
+
+@validataclass
+class ExternalIdentifier(ValidataclassMixin):
+    type: ExternalIdentifierType = EnumValidator(ExternalIdentifierType)
+    value: str = StringValidator()
+
+
+@validataclass
+class KienzlerGeojsonFeaturePropertiesInput(ValidataclassMixin):
+    uid: str = StringValidator(min_length=1, max_length=256)
+    address: Optional[str] = StringValidator(max_length=512), Default(None)
+    type: Optional[ParkingSiteType] = EnumValidator(ParkingSiteType), Default(None)
+    max_height: int = IntegerValidator(min_value=0)
+    max_width: int = IntegerValidator(min_value=0)
+    max_depth: int = IntegerValidator(min_value=0)
+    park_and_ride_type: list[ParkAndRideType] = ListValidator(EnumValidator(ParkAndRideType))
+    external_identifiers: Optional[list[ExternalIdentifier]] = ListValidator(DataclassValidator(ExternalIdentifier))
+
+
+@validataclass
+class KienzlerStaticParkingSiteInput(BaseParkingSiteInput):
```

Comment:
I do wonder why we need that class at all. In the end, you put data in the class at `to_static_parking_site_input()`, and make a dict again at `kienzler_parking_site.from_dict(static_geojson_parking_site_inputs_by_uid[parking_site_uid].to_dict())`. Why not simplify it by renaming `to_static_parking_site_input' to `to_static_parking_site_input_dicts`, return a dict there like this: `return {'lat': self.geometry.coordinates[1],'lon': self.geometry.coordinates[0],**properties_dict}`, `static_geojson_parking_site_inputs_by_uid` gets a `dict[str, dict]` then, which means you can load the dict directly at `from_dict`: `kienzler_parking_site.from_dict(static_geojson_parking_site_inputs_by_uid[parking_site_uid])`? Or do I miss something why `KienzlerStaticParkingSiteInput` is needed?

Reply:
How did you notice this? :O